src/python/arcor2_arserver/rpc/project.py

```diff
@@ -1020,6 +1113,9 @@ async def remove_action_cb(req: srpc.p.RemoveAction.Request, ui: WsClient) -> No
     ap, action = glob.PROJECT.action_point_and_action(req.args.id)
     check_action_usage(action)
 
+    if not await glob.LOCK.is_write_locked(req.args.id, "ui.owner"):
```

Comment:
Tady pak nastává docela zajímavá situace, kdy mám zámek na něco, co už neexistuje. Možná by se to po smazání mohlo samo odemknout? Voláním nějaké funkce v rámci callbacku?

Reply:
To nastává u všech mazacích RPC a obdobně u přiřazení rodiče globálnímu AP (kdy je AP odemčen). Otevřel jsem tohle téma s Michalem Kapinusem, jestli to bude řešit server či editor.